### ISSUE TITLE
Replace jqPlot with another library

### ISSUE THREAD

**Issue by DaSchTour** [matomo-org/matomo#4969]
I think jqPlot should be replaced with another library that is actively developed and supports retina displays.
The problems with retina display are known to jqPlot since 1,5 years and there was no further development to jqPlot since more than a year. So remaining at jqPlot will make piwik worser every day.

**Comment by achiinto** [comment]
Like to see if there is a good graphs library selected for.

**Comment by Findus23** [comment]
Maybe we can close this in favor of #16120, which is newer.
